node/src/components/consensus/highway_core/highway.rs

```diff
@@ -87,6 +87,12 @@ impl<C: Context> From<PreValidatedVertex<C>> for Vertex<C> {
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub(crate) struct ValidVertex<C: Context>(pub(super) Vertex<C>);
 
+impl<C: Context> ValidVertex<C> {
+    pub(crate) fn vertex(&self) -> &Vertex<C> {
```

Comment:
Elsewhere in the code we're using `inner` for things like this. Maybe we can stick to it if you don't mind.